django-test-suite: admin_changelist test_multiuser_edit failure

- [ ] test_multiuser_edit

fails currently with
```
======================================================================
FAIL: test_multiuser_edit (admin_changelist.tests.ChangeListTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/emmanuelodeke/Desktop/spanner-orm/django_tests/django/tests/admin_changelist/tests.py", line 760, in test_multiuser_edit
    self.assertContains(response, '1 swallow was changed successfully.')
  File "/Users/emmanuelodeke/Desktop/spanner-orm/django_tests/django/django/test/testcases.py", line 454, in assertContains
    self.assertTrue(real_count != 0, msg_prefix + "Couldn't find %s in response" % text_repr)
AssertionError: False is not true : Couldn't find '1 swallow was changed successfully.' in response
```

--- a/django_spanner/features.py
+++ b/django_spanner/features.py
@@ -121,8 +121,6 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         # extract() with timezone not working as expected:
         # https://github.com/orijtech/spanner-orm/issues/191
         'timezones.tests.NewDatabaseTests.test_query_datetimes',
-        # To be investigated: https://github.com/orijtech/spanner-orm/issues/135
-        'admin_changelist.tests.ChangeListTests.test_multiuser_edit',
         # using NULL with + crashes: https://github.com/orijtech/spanner-orm/issues/201
         'annotations.tests.NonAggregateAnnotationTestCase.test_combined_annotation_commutative',
         # Spanner loses DecimalField precision due to conversion to float:

